libweb3jsonrpc/IpcServerBase.h

```diff
@@ -46,7 +47,7 @@ template <class S> class IpcServerBase: public jsonrpc::AbstractServerConnector
 	void GenerateResponse(S _connection);
 
 protected:
-	bool m_running = false;
+	std::atomic<bool> m_running{false};
```

Comment:
Let's use then `std::atomic::exchange()` in `IpcServerBase::StartListening` & `IpcServerBase::StopListening` instead of if+assignment

Reply:
That makes sense.  Will do.

Reply:
Done.